slice/IceGrid/Admin.ice

```diff
@@ -591,7 +591,7 @@ module IceGrid
         /// message doesn't exceed the given size.
         /// @param lines The lines read from the file. If there was nothing to read from the file since the last call to
         /// read, an empty sequence is returned. The last line of the sequence is always incomplete (and therefore no
-        /// '\n' should be added when writing the last line to the to the output device).
+        /// new-line character should be added when writing the last line to the to the output device).
```

Comment:
Mapping the `\n` correctly to sphinx requires double escaping when it appears in regular text, or escaping when in a code block.

Reply:
newline in one word